docs/getting-started.md

```diff
@@ -1,21 +1,21 @@
 # Getting started
 
 The tool of choice to get started writing and running workflows is
-the [Symphony generator](https://github.com/SymphonyPlatformSolutions/generator-symphony). Essentially we are going to
```

Comment:
Same changes can be done in README.md:
Lines: 
https://github.com/finos/symphony-wdk/pull/94/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R46

https://github.com/finos/symphony-wdk/pull/94/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R49

Reply:
Good catch